.github/workflows/deploy_branch.yml

```diff
@@ -5,16 +5,29 @@ on:
     pull_request:
 
 jobs:
-  deploy_staging_branch:
-    name: Deploy staging branch
-    uses: zooniverse/Panoptes-Front-End/.github/workflows/azure_deploy.yml@production-release
+  build:
+    name: Build staging branch
+    uses: zooniverse/Panoptes-Front-End/.github/workflows/npm_build.yml@reusable-workflows
```

Comment:
Will the `@reusable-workflows` branch name need to change after this PR is in and the branch cleaned up?

Reply:
Yeah, once this is deployed the refs will need to change to `production_release`.

Reply:
Leaving this here as a reminder to myself to change that ref before I delete this branch.